src/Tribe/Field.php

```diff
@@ -443,6 +443,11 @@ public function do_field() {
 					$tags['option']   = $common_attributes;
 					$tags['fieldset'] = _wp_add_global_attributes( [] );
 
+					// Allow the script tag for HTML fields (inserting script localization).
+					if ( $this->type === 'html' || $this->type === 'wrapped_html' ) {
+						$tags['script'] = true;
```

Comment:
This likely needs the `_wp_add_global_attributes()` call as well.

```suggestion
						$tags['script'] = _wp_add_global_attributes( [] );
```